code/__DEFINES/{yogs_defines}/preferences.dm

```diff
@@ -4,8 +4,9 @@
 #define CHAT_LOOC	(1<<11)
 #define GHOST_CKEY	(1<<12)
 #define CHAT_TYPING_INDICATOR (1<<13)
+#define HEAR_TTS (1<<14)
 #undef TOGGLES_DEFAULT_CHAT
-#define TOGGLES_DEFAULT_CHAT (CHAT_OOC|CHAT_DEAD|CHAT_GHOSTEARS|CHAT_GHOSTSIGHT|CHAT_PRAYER|CHAT_RADIO|CHAT_PULLR|CHAT_GHOSTWHISPER|CHAT_GHOSTPDA|CHAT_GHOSTRADIO|CHAT_BANKCARD|CHAT_LOOC|GHOST_CKEY|CHAT_TYPING_INDICATOR)
+#define TOGGLES_DEFAULT_CHAT (CHAT_OOC|CHAT_DEAD|CHAT_GHOSTEARS|CHAT_GHOSTSIGHT|CHAT_PRAYER|CHAT_RADIO|CHAT_PULLR|CHAT_GHOSTWHISPER|CHAT_GHOSTPDA|CHAT_GHOSTRADIO|CHAT_BANKCARD|CHAT_LOOC|GHOST_CKEY|CHAT_TYPING_INDICATOR|HEAR_TTS)
```

Comment:
make sure this works and doesn't overflow

ALSO

probably needs to be off by default

Reply:
this is the list of toggles turned on by default

Reply:
in latest commit [tatax broke my bitflags 0e83962](https://github.com/yogstation13/Yogstation/pull/14775/commits/0e83962ca73176816c8b4e32156aec10a6def64e) i deleted my player save and started fresh and it works (starts off, needs to be toggled on to hear TTS)

Reply:
running a quick test with it removed

Reply:
ok i think i know whats wrong

Reply:
fixed in last commit